monkey/infection_monkey/island_api_client/http_island_api_client.py

```diff
@@ -81,6 +88,19 @@ def __init__(
         self._session = requests.Session()
         retry_config = Retry(retries)
         self._session.mount("https://", HTTPAdapter(max_retries=retry_config))
+        self._add_request_functions()
+
+    def _add_request_functions(self):
+        for request_method in RequestMethod:
+            request_method_name = str.lower(request_method.name)
+
+            def __get_request_sending_function(method: RequestMethod) -> Callable:
```

Comment:
Why double underscore?

Reply:
This is the `private` notation. I'm not sure what to do in this case, but since it's not accessible from outside scope I'll remove it